department-of-veterans-affairs/va.gov-cms
# VACMS-1487 vet center status import

## Description

See #1487  

## Testing done


## Screenshots


## QA steps

1. Check the status of the migration 
`drush migrate-status va_node_facility_vet_centers_status`  
- [x] Validate that it returns (after several minute wait) :
```
 Group: facility (facility)           Status  Total  Imported  Unprocessed  Last imported 
 va_node_facility_vet_centers_status  Idle    320    0         320            
```
The exact number is 320 based on imported csv file.

2. Run the migration as an update migration 
`drush migrate-import va_node_facility_vet_centers_status --update`

3. The migration may take a while, but after it is completed check the status:
`drush migrate-status va_node_facility_vet_centers_status`  

- [x]  Validate that the numbers look exactly like:
```
 Group: facility (facility)           Status  Total  Imported  Unprocessed  Last imported       
 va_node_facility_vet_centers_status  Idle    320    320       0            2020-04-17 07:18:09
```

4. Visit `/vet-centers/locations/logan-outstation` 
- [x] Validate that it has an operating status `Facility closed` and a status info message `We've closed our Vet Center at this time, following coronavirus COVID-19 guidance from the CDC and our local health department. We're conducting all appointments virtually. Please contact us by phone or online. If you need to talk with someone confidentially, please call us anytime 24/7 at 877-927-8387.`,  and that the page is unpublished. 
- [x] Validate that the last updated date reflects when you ran the migration.

5. Visit `/vet-centers/locations/aberdeen-outstation` 
- [x] Validate that it has an operating status "Limited services and hours" and a status info message `We're currently open for limited in-person service, and screening all visitors for symptoms, due to the coronavirus COVID-19. For individual and group counseling, we recommend using our telehealth services. If you need to talk with someone confidentially, please call us anytime 24/7 at 877-927-8387.`,  and that the page is unpublished.
- [x] Validate that the last updated date reflects when you ran the migration.
- [x] Validate that the revison log shows a revison by "CMS Migrator" and a message indicating it was a status update.


## Definition of Done
